docs/src/components/Header/Header.js

```diff
@@ -6,6 +6,13 @@ import { Link as MDSLink } from '@innovaccer/design-system';
 
 const Header = ({ relativePagePath }) => {
   const items = useHeaderItems();
+  const checkactive = (label) => {
+    let res = false;
+    const pagePath = relativePagePath.split('/');
+    if(pagePath.length>1 && pagePath[1]===label.toLowerCase()) res = true;
+    if(pagePath.length>2 && pagePath[2]===label.toLowerCase()) res = true; 
+    return res; 
+  }
```

Comment:
We can reduce the number of IF statements by combining both conditions. 
```
const checkActive = (label) => {
    const pagePath = relativePagePath.split('/');

    if (pagePath[1] === label.toLowerCase() || pagePath[2] === label.toLowerCase()) {
      return true;
    }
    return false;
  }
```

Reply:
done